legate/core/runtime.py

```diff
@@ -880,7 +1286,273 @@ def dispatch(self, op, redop=None):
         else:
             return op.launch(self.legion_runtime, self.legion_context)
 
-    def _schedule(self, ops):
+
```

Comment:
The prefix sums (for doing bookkeeping of the fused ops) are generated here. 
Given op 'i' in a fusable window, its particular inputs will be located between offsets
inputs_starts[i] and input_starts[i+1] within the inputs list